### ISSUE TITLE
Add Timezone

### ISSUE THREAD

**Issue by shakti97** [mozilla/voice-web#2685]
## Description
We are currently tracking user activities and saving in UTC, that's awesome , But displaying them in UTC as well make the user(who doesn't have knowledge of utc) think that there might be something wrong. So if we could convert the UTC time to local time than there will be no such confusion.

## Enhancement
Add Timezone in the application, to show local time instead of UTC time.

## Additional Information
@phirework  I would like to know your opinion over this.

**Comment by shakti97** [comment]
Hi while checking this enhancement, I found the API for fetching contribution activity is failing for custom locale
![Screenshot from 2020-04-30 03-38-00](https://user-images.githubusercontent.com/30585570/80651776-2643ec80-8a94-11ea-970f-134c3671eb57.png)

can you please check it @phirework , API failing is an issue, whether we will have this feature or not in the future.

**Comment by rileyjshaw** [comment]
@shakti97 good catch! We would accept a pull request for the timezone piece – I think it's best to make the conversion to local time in `web/src/components/pages/dashboard/stats/contribution-activity.tsx`.

As for the API, I'm not sure if I understand correctly. If you're editing the locale parameter with `from=you`, an error is expected. Your `client_id` and `locale` are passed to `getContributionStats()` in `server/src/lib/model/db.ts`. You can see there that unexpected mismatches will cause an error.

Please let me know if I'm misunderstanding the question :) and thanks for all your contributions!

**Comment by moz-bozden** [comment]
Currently the UI shows user's timezone and there are no errors - unless thee is a communication disruption between our servers and the users.
Closing this. Please open a new issue if you encounter any problems.
